### PR TITLE
Port Embeddable JPA tests to OpenLiberty

### PR DESCRIPTION
**Pull request by dazey3**
for #6321

Signed-off-by: Will Dazey <dazeydev.3@gmail.com>

### CODE REVIEW

**Comment by dazey3** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jpa.tests.spec10.embeddable_fat

File: dev/com.ibm.ws.jpa.tests.spec10.embeddable_fat/test-applications/embeddable/nested/src/com/ibm/ws/jpa/embeddable/nested/model/Embeddable01.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
+package com.ibm.ws.jpa.embeddable.nested.model;
```
Comment (jgrassel) [inline dev/com.ibm.ws.jpa.tests.spec10.embeddable_fat/test-applications/embeddable/nested/src/com/ibm/ws/jpa/embeddable/nested/model/Embeddable01.java:12]: Missing the standard header, this is the case for all of these model classes by the looks of it.

File: dev/com.ibm.ws.jpa.tests.spec10.embeddable_fat/test-applications/embeddable/nested/web/embeddableNestedWeb.war/WEB-INF/classes/META-INF/orm-nested.xml [hunk 2]
```diff
@@ -0,0 +1,440 @@
+<entity-mappings xmlns="http://java.sun.com/xml/ns/persistence/orm" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (jgrassel) [inline dev/com.ibm.ws.jpa.tests.spec10.embeddable_fat/test-applications/embeddable/nested/web/embeddableNestedWeb.war/WEB-INF/classes/META-INF/orm-nested.xml:14]: Missing headers (IBM copyright, license, etc) here too.

File: dev/com.ibm.ws.jpa.tests.spec10.embeddable_fat/test-applications/embeddable/relationship/ejb/embeddableRelationshipEjb.jar/META-INF/orm-relationship.xml [hunk 3]
```diff
@@ -0,0 +1,236 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jgrassel) [inline dev/com.ibm.ws.jpa.tests.spec10.embeddable_fat/test-applications/embeddable/relationship/ejb/embeddableRelationshipEjb.jar/META-INF/orm-relationship.xml:1]: Missing headers

File: dev/com.ibm.ws.jpa.tests.spec10.embeddable_fat/test-applications/embeddable/relationship/src/com/ibm/ws/jpa/embeddable/relationship/model/BiM2MOwnerEmbed.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
+package com.ibm.ws.jpa.embeddable.relationship.model;
```
Comment (jgrassel) [inline dev/com.ibm.ws.jpa.tests.spec10.embeddable_fat/test-applications/embeddable/relationship/src/com/ibm/ws/jpa/embeddable/relationship/model/BiM2MOwnerEmbed.java:12]: Model types here also missing headers

File: dev/com.ibm.ws.jpa.tests.spec10.embeddable_fat/test-applications/embeddable/relationship/web/embeddableRelationshipWeb.war/WEB-INF/classes/META-INF/orm-relationship.xml [hunk 5]
```diff
@@ -0,0 +1,236 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jgrassel) [inline dev/com.ibm.ws.jpa.tests.spec10.embeddable_fat/test-applications/embeddable/relationship/web/embeddableRelationshipWeb.war/WEB-INF/classes/META-INF/orm-relationship.xml:1]: Missing header

**Review by jgrassel — CHANGES_REQUESTED** [review]
Many files are missing headers.  Also, I didn't see changes to any of the .github/test-categories/JPA* files, which you need to add your new test bucket to one of them.

**Review by jgrassel — APPROVED** [review]

**Comment by dazey3** [comment]
#run-libby-bot

**Pull request merged**